modules/30-jinx-helpers.sh

```diff
@@ -33,12 +33,22 @@ fi
 # Various useful getter functions
 
 jinx_get_yesterday_timestamp() {
-    if [[ "$OSTYPE" == "freebsd" ]] || [[ "$OSTYPE" == "darwin"* ]]
-    then
-        # for FreeBSD and, maybe, MacOS X
-        date -v -1d +%s
-    else
+    case $OSTYPE in
+        linux)
+            date +%s --date='-1 day'
+        ;;
+        alpine)
+            echo $(($(date +%s) - 86400))
```

Comment:
This version works just as well on GNU/Linux, and perhaps on all the platforms used. Wouldn't it be better to use it in all cases?

Reply:
Personally, I don’t like the mechanism of such a check at all, because during one day, the software can have several security fixes.
Probably, it is necessary to add the check frequency option to the jinx config. And the values of this variable: 0 - do not check and 24h - once a day, as now.